python_src/src/lamp_py/performance_manager/gtfs_utils.py

```diff
@@ -67,7 +67,6 @@ def start_time_to_seconds(
 
 def add_fk_static_timestamp_column(
     events_dataframe: pandas.DataFrame,
-    timestamp_column: str,
```

Comment:
We're not basing GTFS-static matching on a RT-timestamp column anymore, it's only based on `start_date`